File: python/Human.py
# Membuat class bernama Human
class Human:
    
    # Inisialisasi variabel untuk setiap objek dari kelas
    NIK = ""        # Membuat variabel untuk nomor induk kependudukan
    name = ""       # Membuat variabel untuk nama
    gender = ""     # Membuat variabel untuk jenis kelamin
    
    # Konstruktor untuk inisialisasi variabel pada objek dari kelas Human
    def __init__(self, NIK = "", name = "", gender = ""):
        self.NIK = NIK          # Menginisialisasi variabel NIK
        self.name = name        # Menginisialisasi variabel name
        self.gender = gender    # Menginisialisasi variabel gender
        
    # Getter dan Setter untuk NIK
    def get_NIK(self):
        return self.NIK
    
    def set_NIK(self, NIK):
        self.NIK = NIK
        
    # Getter dan Setter untuk Name
    def get_Name(self):
        return self.name
    
    def set_Name(self, name):
        self.name = name
        
    # Getter dan Setter untuk Jenis Kelamin
    def get_Gender(self):
        return self.gender
    
    def set_Gender(self, gender):
        self.gender = gender

File: cpp/Human.cpp
#include <iostream>
#include <string>

/*
    Saya Talitha Syahla NIM 2101330 mengerjakan Soal Latihan 4 
    dalam mata kuliah Desain Pemrograman Berorientasi Objek 
    untuk keberkahanNya maka saya tidak melakukan kecurangan 
    seperti yang telah dispesifikasikan. Aamiin.
*/

using namespace std;

class Human {
    
    private:
        string NIK;     // Membuat variabel untuk nomor induk kependudukan
        string name;    // Membuat variabel untuk nama
        string gender;  // Membuat variabel untuk jenis kelamin

    public:
        // Konstruktor untuk inisialisasi variabel pada objek dari kelas Human
        Human(){
            this->NIK = "";         // Membuat variabel untuk nomor induk kependudukan
            this->name = "";        // Membuat variabel untuk nama
            this->gender = "";      // Membuat variabel untuk jenis kelamin
        }

        Human(string NIK, string name, string gender){
            this->NIK = NIK;
            this->name = name;
            this->gender = gender;
        }

    // Setter dan Getter

        // Getter dan Setter untuk NIK
        string get_NIK() {
            return NIK;
        }

        void set_NIK(string NIK) {
            this->NIK = NIK;
        }

        // Getter dan Setter untuk Name
        string get_Name() {
            return name;
        }

        void set_Name(string name) {
            this->name = name;
        }

        // Getter dan Setter untuk Jenis Kelamin
        string get_Gender() {
            return gender;
        }

        void set_Gender(string gender) {
            this->gender = gender;
        }

        // Destruktor
        ~Human(){
        }
        
};